Extension/package.nls.json

```diff
@@ -194,6 +194,7 @@
     "c_cpp.configuration.simplifyStructuredComments.markdownDescription": { "message": "If `true`, tooltips of hover and auto-complete will only display certain labels of structured comments. Otherwise, all comments are displayed.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
     "c_cpp.configuration.doxygen.generateOnType.description":  "Controls whether to automatically insert the Doxygen comment after typing the chosen comment style.",
     "c_cpp.configuration.doxygen.generatedStyle.description": "The string of characters used as the starting line of the Doxygen comment.",
+    "c_cpp.configuration.doxygen.selectDoxygenTags.description": "Select which doxygen tags you would like to appear in formatted structued comment view.",
```

Comment:
Misspelling of "structued" -> "structured".

There's code spell checker extension for VS Code if you'd like to run a full spell check on files. https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
"doxygen" should be capitalized.